flow/util/template_filters.py

```diff
@@ -356,3 +356,17 @@ def get_account_name(environment, required=False):
                 file=sys.stderr,
             )
             _GET_ACCOUNT_NAME_MESSAGES_SHOWN.add(env_name)
+
+
+def format_operation_name(operation_name, project):
+    """Return the operation name or group name with number of operations."""
+    return (
+        f"{operation_name} ({len(project._groups[operation_name].operations)} ops)"
+        if (
+            # both checks in conditional are necessary to ensure that the status line that prints
+            # out the number of operations omitted passes through without causing errors.
+            operation_name in project._groups
+            and operation_name not in project._operations
+        )
+        else operation_name
+    )
```

Comment:
This logic is correct but it's written awkwardly with comments interspersed within the return statement. I suggest:
```suggestion
    # If the name is from a group that is not an operation, we append the
    # number of operations to its name in the status.
    if operation_name in project._groups and operation_name not in project._operations:
        num_operations = len(project._groups[operation_name].operations)
        return f"{operation_name} ({num_operations} ops)"
    return operation_name
```

Reply:
That is how it was but @csadorf argued for a more functional style.

Reply:
I'm going to apply my suggestion on the grounds that it's fewer lines of code and that it was originally written that way. Resolving.